plover/dictionary/rtfcre_dict.py

```diff
@@ -307,8 +308,9 @@ def load_dictionary(filename):
           "{\\stylesheet{\\s0 Normal;}}\r\n")
 
 def format_translation(t):
-    t = ' '.join([x.strip() for x in META_RE.findall(t) if x.strip()])
```

Comment:
Okay, but in this particular case this is just from JSON translation to RTF, so having the extra whitespace shouldn't be an issue (as \\n \\r and \\t are escaped)